### PR TITLE
fix(doc): fix community vs subscription info and provide examples for upload servlets usage

### PR DESCRIPTION
**Pull request by amottier**
Update documentation for upload servlets as pageUpload and applicationsUpload are available in Community edition.
Add more details about how to use the upload servlets and provide a link to an example (Groovy code) host on bonitasoft-community GitHub organization.

Help to answer: http://stackoverflow.com/questions/43146828/how-to-use-rest-api-to-create-extension-api-on-bonita-bpm

Closes [BS-16533](https://bonitasoft.atlassian.net/browse/BS-16533)

### CODE REVIEW

File: md/manage-files-using-upload-servlet-and-rest-api.md [hunk 1]
```diff
@@ -1,58 +1,40 @@
...
 
-#### Example
-Request url
-/API/identity/group
 
-Request method
+### Link an icon to a new organization group
 
-POST
+1. Call to the login service: `/bonita/loginservice` for authentication.
```
Comment (abirembaut) [inline md/manage-files-using-upload-servlet-and-rest-api.md]: remove the "to" in order to be consistent with the other points (imperative form)

File: md/manage-files-using-upload-servlet-and-rest-api.md [hunk 2]
```diff
@@ -1,58 +1,40 @@
...
-    "creation_date":"2014-12-02 16:19:28.925",
-    "created_by_user_id":"4",
-    "icon":"tmp_7501171996762091204.jpg","parent_path":"/acme"
-    ,"description":"Human resources department",
-    "name":"HR",
-    "path":"/acme/HR",
-    "displayName":"Human Resources",
-    "last_update_date":"2014-12-02 16:19:28.925"
-    }
+1. Call to the login service: `/bonita/loginservice` for authentication.
```
Comment (abirembaut) [inline md/manage-files-using-upload-servlet-and-rest-api.md]: remove the "to"

**Review by abirembaut — COMMENTED** [review]
LGTM just a small comment
A README.md in the github project could be a good idea too. But that doesn't prevent merging this PR

**Comment by danila-m** [comment]
I removed the "to" allow the PR to be merged.

**Pull request merged**